src/_pytest/fixtures.py

```diff
@@ -669,20 +718,21 @@ def _schedule_finalizers(self, fixturedef, subrequest):
         super()._schedule_finalizers(fixturedef, subrequest)
 
 
-scopes = "session package module class function".split()
+scopes = ["session", "package", "module", "class", "function"]  # type: List[_Scope]
 scopenum_function = scopes.index("function")
 
 
-def scopemismatch(currentscope, newscope):
+def scopemismatch(currentscope: "_Scope", newscope: "_Scope") -> bool:
     return scopes.index(newscope) > scopes.index(currentscope)
 
 
-def scope2index(scope, descr, where=None):
+def scope2index(scope: str, descr: str, where: Optional[str] = None) -> int:
     """Look up the index of ``scope`` and raise a descriptive value error
     if not defined.
     """
+    strscopes = scopes  # type: Sequence[str]
```

Comment:
Just a hack to expand the type of `scopes` to allow checking against an `str` rather than `_Scope`.